docs/emr/presto-admin-installation-on-emr.rst

```diff
@@ -0,0 +1,32 @@
+=========================
+Presto Admin Installation 
```

Comment:
I don't think it would be too confusing to put on the main page a note that says something like ("The offline installer is built for a particular operating system environment.  If you are having problems with the offline installer try the online installer <link to online installer & link to page discussing the difference>).